CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [5.1.0]
+### Changed
+- Fix `eth_getFilterLogs` ([#84](https://github.com/MetaMask/eth-json-rpc-filters/pull/84))
+- Fix for null reference exception when normalizing the block ([#83](https://github.com/MetaMask/eth-json-rpc-filters/pull/83))
+- Chore: update eth-json-rpc-middleware to ^9 ([#76](https://github.com/MetaMask/eth-json-rpc-filters/pull/76))
```

Comment:
Two things; these "fixed" entries should be under the "Fixed" category, and for the version bump it's probably better described as a removal because it's a devDependency now.

```suggestion
### Changed
- Remove `eth-json-rpc-middleware` dependency ([#76](https://github.com/MetaMask/eth-json-rpc-filters/pull/76))
  - This package is still used in tests as a `devDependency`, but it's not used in the published package anymore.

### Fixed
- Fix `eth_getFilterLogs` ([#84](https://github.com/MetaMask/eth-json-rpc-filters/pull/84))
- Fix for null reference exception when normalizing the block ([#83](https://github.com/MetaMask/eth-json-rpc-filters/pull/83))
```